aiven/resource_project_test.go

```diff
@@ -138,6 +138,7 @@ func testAccProjectResource(name string) string {
 			default_cloud = "aws-eu-west-2"
 			billing_currency = "EUR"
 			vat_id = "123"
+			card_id = ""
```

Comment:
I reckon we should just remove `resource.TestCheckResourceAttrPair(datasourceName, "card_id", resourceName, "card_id"),` from the checks, since it is not really checking anything, perhaps we can add more meaningful check-ins instead validating other attributes.

Reply:
Removed that line from the checks. I agree with the meaningful checks and maybe we can do that in a separate PR.